app/helpers/sufia/sufia_helper_behavior.rb

```diff
@@ -123,6 +123,20 @@ def index_field_link(options)
       safe_join(links, ", ")
     end
 
+    # A Blacklight helper_method
+    # @param [Hash] options from blacklight invocation of helper_method
+    # @return the html_safe facet links separated by " > " to indicate facet hierarchy
+
+    def link_to_each_facet_field(options)
+      raise ArgumentError unless options[:config] && options[:value] && options[:config][:helper_facet]
+      facet_search = options[:config][:helper_facet]
+      facet_fields = options[:value].first.split(">").each(&:strip!)
```

Comment:
@cbeer is there a way of configuring the hierarchical separator that we could use here?